### PR TITLE
add ascend op for VIT&&YOLOv3

### PR DESCRIPTION
**Pull request by hellozmz**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation and Context

add ascend op for VIT&&YOLOv3

## Description

add ascend op for VIT&&YOLOv3

## Checklist
**Before PR**:

- [ ] I have read and followed the workflow indicated in the [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) to create this PR.
- [ ] Pre-commit or linting tools indicated in [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: impl/ascend/common/acloprunner.hpp [hunk 1]
```diff
@@ -163,6 +163,14 @@ diopiError_t makeTensorLike(diopiContextHandle_t ctx, diopiTensorHandle_t* out,
 
 diopiError_t makeTensorLike(diopiContextHandle_t ctx, diopiTensorHandle_t* out, diopiConstTensorHandle_t src, diopiDtype_t dtype);
 
+/**
+ * @brief some op originally support positive tensor, but ascend op can handle negative tensor. So we need to change those out value to nan
+ * @param[in] ctx Context environment.
+ * @param[in] input the input tensor.
+ * @param[out] the output tensor.
+ */
+diopiError_t negativeInputRtnFillNan(diopiContextHandle_t ctx, diopiTensorHandle_t out, diopiConstTensorHandle_t input);
```
Comment (chrysantd) [inline impl/ascend/common/acloprunner.hpp:172]: 我看了下，你这个实现的功能很小，跟你的算子很相关，别放acloprunner和utils里面吧，放算子的文件里

File: impl/ascend/functions/tanh.cpp [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+#include <diopi/functions.h>
+
+#include "../common/acloprunner.hpp"
+
+namespace impl {
+namespace ascend {
+
+extern "C" {
+
+DIOPI_API diopiError_t diopiTanh(diopiContextHandle_t ctx, diopiTensorHandle_t out, diopiConstTensorHandle_t input) {
```
Comment (chrysantd) [inline impl/ascend/functions/tanh.cpp:16]: 这个放activation.cpp里

File: impl/ascend/functions/activation.cpp [hunk 3]
```diff
@@ -58,5 +58,28 @@ extern "C" DIOPI_API diopiError_t diopiSigmoidBackward(diopiContextHandle_t ctx,
...
+    return diopiSuccess;
+}
+
+extern "C" DIOPI_API diopiError_t diopiLeakyRelu(diopiContextHandle_t ctx, diopiTensorHandle_t out, diopiConstTensorHandle_t input,
+                                                 const diopiScalar_t* negative_slope) {
+    AclOpRunner<1, 1>("LeakyRelu", ctx).addInput(input).setAttr("negative_slope", getValue<float>(negative_slope)).addOutput(out).run();
+    return diopiSuccess;
+}
+
+extern "C" DIOPI_API diopiError_t diopiLeakyReluBackward(diopiContextHandle_t ctx, diopiTensorHandle_t grad_input, diopiConstTensorHandle_t grad_output,
```
Comment (chrysantd) [inline impl/ascend/functions/activation.cpp:78]: 没开格式检查吗，变量名应该改成驼峰式的

File: impl/ascend/functions/tanh.cpp (hunk 2 above)
Comment (hellozmz) [reply impl/ascend/functions/tanh.cpp]: 改了

**Review by chrysantd — APPROVED** [review]

**Pull request merged**